src/wp-admin/includes/class-wp-ms-sites-list-table.php

```diff
@@ -702,19 +702,19 @@ protected function handle_row_actions( $item, $column_name, $primary ) {
 		$actions['edit']    = '<a href="' . esc_url( network_admin_url( 'site-info.php?id=' . $blog['blog_id'] ) ) . '">' . __( 'Edit' ) . '</a>';
 		$actions['backend'] = "<a href='" . esc_url( get_admin_url( $blog['blog_id'] ) ) . "' class='edit'>" . __( 'Dashboard' ) . '</a>';
 		if ( get_network()->site_id != $blog['blog_id'] ) {
-			if ( '1' == $blog['deleted'] ) {
+			if ( '1' === $blog['deleted'] ) {
```

Comment:
pls revert this condition if you have already opened PR

```suggestion
            if ( $blog['deleted'] === '1' ) {

Reply:
@Jankyz WordPress sticks to the Yoda condition throughout the code. I think my initial commit stands.

Reply:
Yes of course, but Yoda's condition is an old-fashion way :) Anyway, it was only a suggestion to turn WordPress on the modern way.